src/components/send/SendContactList.js

```diff
@@ -129,13 +129,13 @@ export default function SendContactList({
   const filteredAddresses = useMemo(() => {
     return sortBy(
       filterList(
-        userAccounts.filter(account => account.visible && account.address.toLowerCase() !== accountAddress.toLowerCase()),
+        userAccounts.filter(account => account.visible),
```

Comment:
was this change intentional? If I'm not mistaken we would still want to filter out the users address so they can't send to themselves.